packages/web/src/apollo/index.tsx

```diff
@@ -55,6 +78,8 @@ export type GraphQLClientConfigProp = Omit<
    */
   httpLinkConfig?: apolloClient.HttpOptions
   /**
+   * @deprecated
+   * @see linkFactory
```

Comment:
I opted to deprecate `link` as a whole so it's less confusing about which is preferred & how it might behave if you provide both props

Thoughts?

`linkFactory: () => mySingletonLink`

isn't any harder than
`link: mySingletonLink`

Reply:
I think this is the right move, the new API is better and isn't that much harder by any means. I think it's pretty rare that users would want to just completely overwrite the link